src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.ts

```diff
@@ -219,6 +220,11 @@ export class CreateEditEndpointComponent
       this.urlDisabled = false;
       this.targetForm.controls.endpointUrl.setValue("");
     }
+    if (selectValue === 'google-gcr') {
+      this.targetForm.controls.access_key.setValue("_json_key");
+    }else{
```

Comment:
it display formatting problems in travels CI

Reply:
fixed